### DISCUSSION TITLE
I have a problem about new version

### DISCUSSION THREAD

**Discussion by urdkr** [Bing-su/adetailer#421] [Q&A]
I updated your program. After that, I got a problem about adetailer. When I used adetailer, I made below photos.
I am using majicmixRealistic_v7.
 
<img width="200" alt="2023-11-20 112013" src="https://github.com/Bing-su/adetailer/assets/58834406/297d2632-90a1-4c46-81b4-b93f2d1e605e">

**Comment by Argon5504** [comment]


**Reply by urdkr** [reply]
Thank you very much for you answer!!
very helpful
